posthog/models/error_tracking/sql.py

```diff
@@ -81,3 +86,97 @@
 """
+
+ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE = "error_tracking_issue_fingerprint_embeddings"
+ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_WRITABLE_TABLE = f"writable_{ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE}"
+KAFKA_ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE = f"kafka_{ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE}"
+ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_MV = f"{ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE}_mv"
+
+ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} {on_cluster_clause}
+(
+    team_id Int64,
+    model_name LowCardinality(String),
+    embedding_version Int64, -- This is the given iteration of the embedding approach - it will /probably/ always be 0, but we want to be able to iterate on e.g. what we feed the model, so we'll leave that door open for now
+    fingerprint VARCHAR,
+    inserted_at DateTime64(3, 'UTC'),
+    embeddings Array(Float64) -- We could experiment with quantization, but if we do we can use a new column, for now we'll eat the inefficiency
+    {extra_fields} -- Unused, I think, but the above has it, so
+) ENGINE = {engine}
+"""
+
+
+def ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_ENGINE():
+    return ReplacingMergeTree(ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE, ver="inserted_at")
+
+
+def ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_SQL(on_cluster=True):
+    return (
+        ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_BASE_SQL
+        + """
+    ORDER BY (team_id, model_name, embedding_version, fingerprint)
+    SETTINGS index_granularity = 512
+    """
+    ).format(
+        table_name=ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE,
+        on_cluster_clause=ON_CLUSTER_CLAUSE(on_cluster),
+        engine=ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_ENGINE(),
+        extra_fields=f"""
+    {KAFKA_COLUMNS_WITH_PARTITION}
+    , {index_by_kafka_timestamp(ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE)}
+    """,
+    )
+
+
+def ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_WRITABLE_TABLE_SQL():
+    return ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_BASE_SQL.format(
+        table_name=ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_WRITABLE_TABLE,
+        on_cluster_clause=ON_CLUSTER_CLAUSE(False),
+        engine=Distributed(
+            data_table=ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE,
+            cluster=settings.CLICKHOUSE_SINGLE_SHARD_CLUSTER,
+        ),
+        extra_fields=KAFKA_COLUMNS_WITH_PARTITION,
+    )
+
+
+def KAFKA_ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_SQL(on_cluster=True):
+    return ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE_BASE_SQL.format(
+        table_name=KAFKA_ERROR_TRACKING_FINGERPRINT_EMBEDDINGS_TABLE,
+        on_cluster_clause=ON_CLUSTER_CLAUSE(on_cluster),
+        engine=kafka_engine(
+            KAFKA_ERROR_TRACKING_ISSUE_FINGERPRINT_EMBEDDINGS, group="clickhouse-error-tracking-fingerprint-embeddings"
```

Comment:
This is a silly request, but if this is a new table, could we use `_` instead? We have a chaotic mix of group names, and we'll probably soon refactor those so they are all `clickhouse_whatever` which will allow us to ease their monitoring. 🙏 

```suggestion
            KAFKA_ERROR_TRACKING_ISSUE_FINGERPRINT_EMBEDDINGS, group="clickhouse_error_tracking_fingerprint_embeddings"
```